runtime/vm/vmthinit.c

```diff
@@ -80,6 +80,8 @@ UDATA initializeVMThreading(J9JavaVM *vm)
 
 		omrthread_monitor_init_with_name(&vm->vmRuntimeStateListener.runtimeStateListenerMutex, 0, "VM state notification mutex") ||
 
+		omrthread_monitor_init_with_name(&vm->constantDynamicMutex, 0, "Lock for Constant Dynamic resolve store") ||
```

Comment:
This is a single lock shared by all threads and all constantdynamics.  We should be able to resolve without a lock